src/library/base/man/paste.Rd

```diff
@@ -120,18 +121,19 @@ paste(month.abb, nth, sep = ": ", collapse = "; ")
     width = 30), collapse = "\n"))
 plot(dist ~ speed, cars, main = title)
 
-## 'recycle0 = TRUE' allows more vectorized behaviour, i.e. zero-length recycling :
-valid <- FALSE
-val <- pi
-paste("The value is", val[valid], "-- not so good!")
-paste("The value is", val[valid], "-- good: empty!", recycle0=TRUE) # -> character(0)
+# zero length arguments recycled as `""`
+paste({}, 1:2)
```

Comment:
Personally I would prefer `paste(character(), 1:2)`. `{}` is concise, but masks two behaviors:

 1. the empty expression evaluates to `NULL`: `identical(NULL, {})` is `TRUE`
 2. [inputs to `paste()` are cast with `as.character()`](https://github.com/r-devel/r-svn/blob/3c24c211df69cf01c61ec61e5de5ad053cdf1f1c/src/main/paste.c#L147-L161)
 
 Using `character()` is thus more "minimal" in the sense of demonstrating the behavior more directly

Reply:
Thank you for pointing this out, @MichaelChirico!